src/app/screens/Game/components/Board/index.js

```diff
@@ -1,32 +1,30 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square/index.js';
 
-import style from './styles.css';
+import './styles.css';
 
 class Board extends Component {
   // eslint-disable-next-line
   renderSquare(i) {
-    return <Square />;
+    return <Square value={this.props.squares[i]} onClick={() => this.props.onClick(i)} />;
```

Comment:
to avoid instance a new function arrow on each render you can pass the `i` param to Square component. Then Square component should has a function like `handleClick` that execute onClick function using the `i` prop.

Reply:
Done